Pseudocode/Sorting/Quick Sort/SourceCode.tex

```diff
@@ -0,0 +1,66 @@
+% Set the Page Layout
+\documentclass[12pt]{article}
+\usepackage[inner = 2.0cm, outer = 2.0cm, top = 2.0cm, bottom = 2.0cm]{geometry}
+
+% Package to write pseudo-codes
+\usepackage{algorithm}
+
+% Don't Remove the 'end' at the end of the algorithm
+\usepackage{algpseudocode}
+
+% Manually remove the 'end' for some sections
+\algtext*{EndIf}
+\algtext*{EndFunction}
```

Comment:
Let's keep the visual `EndFunction` for this problem. (As you did for the `EndProcedure`). To accomplish that, delete this line.